tests/snippets/division_by_zero.py

```diff
@@ -0,0 +1,34 @@
+try:
+    5 / 0
+except ZeroDivisionError:
+    pass
+except:
+    assert False, 'Expected ZeroDivisionError'
+
+try:
+    5 / -0.0
+except ZeroDivisionError:
+    pass
+except:
+    assert False, 'Expected ZeroDivisionError'
+
+try:
+    5 / (3-2)
```

Comment:
Why is a `ZeroDivisionError` expected on this?

Reply:
It shouldn’t, it was supposed to be `(2-2)`… The fact that it passes at least caught a flaw in the tests themselves. Will fix them in a moment.